framework/Sources/VGSFramework/Core/APIClient.swift

```diff
@@ -44,7 +43,49 @@ class APIClient {
         ]
     }()
     
+    @available(*, deprecated, message: "use -sendRequest:")
     func sendRequest(path: String, method: HTTPMethod = .post, value: BodyData, completion block: @escaping (_ data: JsonData?, _ error: Error?) -> Void) {
+        // JSON Body
+        let body: [String: Any] = value
+        // Fetch Request
+        sendRequest(path: path, method: method, value: body) { VGSResponse in
+            switch VGSResponse {
+            case .success( _, let data):
+                
+                var resultData: JsonData?
+                if let data = data, let json = try? JSONSerialization.jsonObject(with: data, options: .mutableContainers) as? JsonData {
+                    resultData = json
+                }
+                block(resultData, nil)
+                
+            case .failure(let error):
+                block(nil, error)
+            }
+        }
+    }
+}
+
+// MARK: - URLSession
+public enum VGSResponse {
+    case success(Bool, Data?)
```

Comment:
not sure that we need to send success(False/True), aspesically it can be confusing for developers what that means when request is success but hv False value

Reply:
How about the response which has only status code 200?